catala/catala.md

```diff
@@ -7,12 +7,12 @@ Tanque els ulls els torne a obrir. No pot ser. On estic?
 Em resulta familiar, però no sé on estic. [Que és aixó ](http://www.google.es/imgres?imgurl=http%3A%2F%2F1.bp.blogspot.com%2F-b0CZu7RAyiM%2FULaNOLTJMZI%2FAAAAAAAAAA8%2FwmVTGAKDC-U%2Fs400%2FCaptura3.PNG&imgrefurl=http%3A%2F%2Fsusandsan.blogspot.com%2F2012%2F11%2Fsimbolismo-figuracion-y-abstraccion.html&h=304&w=400&tbnid=jAdgGBhRlYxgwM%3A&zoom=1&docid=j7NOlpg8lyjPMM&ei=xo95VOvhDc7zasWGgaAP&tbm=isch&ved=0CGMQMygfMB8&iact=rc&uact=3&dur=1931&page=3&start=20&ndsp=13)
 
 Què fas?
-[Et poses a dormir] (dormir/dormir.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.